### ISSUE TITLE
GIBCT Estimate Your Benefits - Full Audit and VA 508 Audit

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#13194]
This issue will serve as the epic to store any findings from the [Estimate Your Benefits audit request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10786).

**Comment by 1Copenut** [comment]
@desireeturner I've finally had a chance to run through this audit. I found a few items, but most of them are quick fixes. I have two items that I'm not going to ticket, but will mention to the VA 508 office for their input:

1. Chrome + JAWS 2020 doesn't set focus correctly on the accordion button when I press the "Update benefits" button after making a change to the benefits inputs.
1. IE11 + JAWS 2020 doesn't announce the live-region changes to benefits when users press the "Update benefits" button.

These are both specific to a single browser + screen reader, so they might be implementation bugs. I'll follow up with you when the VA 508 office weighs in.

I"ll plan to send this for a VA 508 office audit in my call with them on Thursday unless I hear differently from you.

Thank you!

**Comment by 1Copenut** [comment]
@desireeturner This feature has been sent to the VA 508 office for their quick test. I"ll follow up if there are any additional items to discuss.

**Comment by desireeturner** [comment]
@1Copenut Closing this epic as all related issues have been resolved.
